sdk/conformance/corpus-gate.sh

```
@@ -130,3 +137,62 @@ if [ "$checked" -eq 0 ]; then
+provenance_fails=0
+provenance_checked=0
+printf "\n%-44s %-12s %-12s %-12s %-12s %s\n" PROVENANCE GO TS RUST PY RESULT
+for f in "$PROVENANCE_CORPUS"/*.json; do
+  [ -f "$f" ] || continue
+  case "$f" in *.expect.json) continue ;; esac
+  base="$(basename "$f" .json)"
+  expfile="$PROVENANCE_CORPUS/$base.expect.json"
+  [ -f "$expfile" ] || { echo "FATAL: missing provenance expectation for $base" >&2; exit 2; }
+  expected="$(tr -d '\n ' < "$expfile")"
+
+  go="$(provenance_output "$f" $GO_PROVENANCE)" || exit 2
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)